CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
+
+TestCafe Live provides a service that keeps the TestCafe process and browsers opened the whole time you are working on tests. Changes you make in code immediately restart the tests. That is, TestCafe Live allows you to see test results instantly.
+
+[![TestCafe Live Video](https://raw.githubusercontent.com/DevExpress/testcafe/master/media/changelog/testcafe-live-video.png)](https://www.youtube.com/watch?v=RWQtB6Xv01Q)
+
+For more information, see [TestCafe Live](https://github.com/DevExpress/testcafe-live).
+
+### Enhancements
+
+#### Taking Screenshots of Individual Page Elements ([#1496](https://github.com/DevExpress/testcafe/issues/1496))
+
+We have added the [t.takeElementScreenshot](https://devexpress.github.io/testcafe/documentation/test-api/actions/take-screenshot.html#take-a-screenshot-of-a-page-element) action that allows you to take a screenshot of an individual page element.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `My fixture`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+test('Take a screenshot of a fieldset', async t => {
+    await t
+        .click('#reusing-js-code')
+        .click('#continuous-integration-embedding')
+        .takeElementScreenshot(Selector('fieldset').nth(1), 'my-fixture/important-features.png');
+});
+```
+
+This action provides additional customization that allows you to position the center of the screenshot or crop it. For more information, see the [documentation](https://devexpress.github.io/testcafe/documentation/test-api/actions/take-screenshot.html#take-a-screenshot-of-a-page-element).
+
+Note that if the screenshot directory is not specified with the [runner.screenshots](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#screenshots) API method or the [screenshots](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-s-path---screenshots-path) command line option, the `t.takeElementScreenshot` action will be ignored.
+
+#### Filtering Elements by Their Visibility ([#1018](https://github.com/DevExpress/testcafe/issues/1018))
+
+You can now filter the selector's matching set to leave only visible or hidden elements. To do this, use the [filterVisible](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.html#filtervisible) and [filterHidden](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.html#filterhidden) methods.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `My fixture`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+test('Filter visible and hidden elements', async t => {
+    const inputs        = Selector('input');
+    const hiddenInput   = inputs.filterHidden();
+    const visibleInputs = inputs.filterVisible();
+
+    await t.expect(hiddenInput.value).eql(1);
```

Comment:
![image](https://user-images.githubusercontent.com/4133518/36430172-8a09c05a-1665-11e8-8b3d-1dfd7a295c41.png)

Reply:
Let's put these aseertions in chain
```js
await t
    .expect('...')
    .expect('...');
```